Resources/Prototypes/DeltaV/Entities/Stations/base.yml

```diff
@@ -0,0 +1,61 @@
+- type: entity
+  id: BaseStationStockMarket
+  abstract: true
+  components:
+  - type: StationStockMarket
+    companies:
+      Nanotrasen:
+        priceHistory:
+        - 100
+        - 100
+        - 100
+        - 100
+        - 100
+        basePrice: 100
+        currentPrice: 100
+        displayName: Nanotrasen [NT]
+        name: Nanotrasen
+      Gorlex:
+        priceHistory:
+        - 75
+        - 75
+        - 75
+        - 75
+        - 75
```

Comment:
have this be in the mapinit handler and when adding a new company to set history to 5 of the current price

Reply:
should already do it on component init and but i had to make this meme because of the test fail 
i'll play around with it more

Reply:
use MapInit not ComponentInit